client/src/components/navigation/Navbar.js

```diff
@@ -94,7 +94,7 @@ const Navbar = props => {
     const primary = breadcrumbNameMap[to];
 
     return (
-      <li className={classes.text}>
+      <li className={classes.navRoutes}>
```

Comment:
Looks like you got the navRoutes name going (or am I getting it backwards and navRoutes isn't what you want).